config-model/src/main/java/com/yahoo/vespa/model/search/IndexedSearchCluster.java

```diff
@@ -369,6 +366,9 @@ public void getConfig(DispatchConfig.Builder builder) {
                     break;
             }
         }
+        if (tuning.dispatch.maxHitsPerPartition != null)
```

Comment:
Please consider "{" on one-line if statements.

Reply:
Normally I do, but I prefer following the consensus in the file. And here all where non-braced.